pkg/gadgets/traceloop/tracer/bpf/traceloop.bpf.c

```diff
@@ -48,6 +48,10 @@
 #define bpf_error_printk(fmt, ...)
 #endif /* !defined(SHOW_ERROR) */
 
+/* These are needed to not make use of cgo. */
```

Comment:
I think this comment is misleading. Even if this is true the final purpose of them is not to use cgo, the immediate reason whey they are here is to avoid the compiler optimizing out  those definitions from the elf file.